docker-compose/cassandra-3.11/docker-compose.yml

```diff
@@ -75,11 +75,10 @@ services:
     networks:
       - stargate
     ports:
-      - 9080:9080
+      - 8085:8080
```

Comment:
I changed the host port because I happen to have something already running on 9080 on my laptop.
Also it aligns better with what is used in the rest of the file.